packages/amplify-headless-interface/schemas/geo/1/AddGeoRequest.schema.json

```diff
@@ -48,9 +48,6 @@
                       "VectorHereBerlin"
                   ]
               },
-              "pricingPlan": {
```

Comment:
these changes should be made in `amplify-headless-interface/src/interface/geo/add.ts` and synthesized here using `yarn generateSchemas --overwrite`

Also do we need to bump the version of the headless paylaod? Seems like this is a breaking interface change

Reply:
Thanks Ed! updated the `amplify-headless-interface/src/interface/geo/add.ts` file